css/cssom/getComputedStyle-pseudo.html

```diff
@@ -91,4 +98,15 @@
                   "Pseudo-styles of display: flex elements should get blockified");
   });
 }, "Item-based blockification of nonexistent pseudo-elements");
+test(function() {
+  var contentsPseudos = document.getElementById('contents-pseudos');
+  [":before", ":after"].forEach(function(pseudo) {
+    assert_equals(getComputedStyle(contentsPseudos, pseudo).display, "contents",
+                  "display: contents in " + pseudo + " should get reflected on CSSOM");
+    assert_equals(getComputedStyle(contentsPseudos, pseudo).borderLeftWidth, "0px",
+                  pseudo + " with display: contents should have no box");
```

Comment:
Isn't 0px here because border-style is "none"? What if you set border-style to "solid"? I couldn't find that the resolved value of border-width should depend on whether a box is generated or not.

Reply:
Err, you're right.

Reply:
Updated to use `width`.